private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/gradle/models/Exclusion.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.github.bazelbuild.rules_jvm_external.resolver.gradle.models;
+
+/**
+ * Exclusion models a local exclusion to be applied to a dependency in build.gradle.kts
+ */
+public interface Exclusion {
```

Comment:
This is one of many interfaces and implementations needed to work with the gradle ToolingModelBuilder api to properly serialize the dependency information that's generated in the plugin.